### DISCUSSION TITLE
dependencies erros while using socket.io-client library

### DISCUSSION THREAD

**Discussion by saikrishna488** [vercel/community#2167] [Bug Report]
socket.io-client library was working perfectly with old pages dir
but has dependencies erros with new app dir

**Comment by saikrishna488** [comment]
![Screenshot 2023-04-23 140943](https://user-images.githubusercontent.com/117559736/233829282-ab22e726-e07a-4fb9-a135-07b6d60495ea.png)

**Comment by amyegan** [comment]
This sounds similar to https://github.com/vercel/next.js/discussions/48422. As a quick test, assuming you are using Next.js `13.3.0` or `13.3.1`, do you still get the same error if you go back to `13.2.4`?

**Reply by saikrishna488** [reply]
i was using 13.3.1 with app dir and after getting those erros i tried same version with src dir everything was working fine
